app/javascript/components/lease_ups/LeaseUpsTable.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import ReactTable from 'react-table'
+import _ from 'lodash'
+
+import DropdownMenuItem  from '../atoms/DropdownMenuItem'
+import Dropdown from '../molecules/Dropdown'
+import PrettyTime from '../utils/PrettyTime'
+
+import { PREFRENCE_RANK_LABELS, LEASE_UP_STATUS_OPTIONS, PAGE_SIZE } from './data'
+
+const PreferenceRankCell = ({ cell }) => {
+  const label = PREFRENCE_RANK_LABELS[cell.value] || cell.value
+  return (<div>{label}</div>)
+}
+
+const LeaseUpStatusCell = ({ cell, onChange }) => {
+  const value = cell.value || ''
+
+  // return (
```

Comment:
If you want to keep this comment for the future story on filtering, please put a comment to reference the context and purpose of it :)